source/_components/light.hue.markdown

```diff
@@ -29,12 +29,22 @@ light:
   host: DEVICE_IP_ADDRESS
 ```
 
+The light component supports multiple entries in <code>configuration.yaml</code> by appending a sequential number to the section: <code>light 2:</code>, <code>light 3:</code> etc.  When adding in a second hub, add the following lines to your `configuration.yaml`:
+
+```yaml
+# Example configuration.yaml entry
+light 2:
+  platform: hue
+  host: DEVICE_IP_ADDRESS
+  filename: phue2.conf
+```
+
 Configuration variables:
 
 - **host** (*Optional*): IP address of the device, eg. 192.168.1.10. Required if not using the `discovery` component to discover Hue bridges.
 
 - **allow_unreachable** (*Optional*): (true/false)  This will allow unreachable bulbs to report their state correctly.
-- **filename** (*Optional*): Make this unique if specifying multiple Hue hubs.
+- **filename** (*Optional*): Make this unique if specifying multiple Hue hubs. Necessary though if adding second Hub.
```

Comment:
I don't think that that the added sentence adds any value compared to what is already there. Instead, rewrite the description to explain what it is used for instead of how it should be used.